tin/apps/assignments/forms.py

```diff
@@ -55,6 +64,31 @@ class Meta:
             "submission_limit_cooldown": "Rate limit cooldown period (minutes)",
             "is_quiz": "Is this a quiz?",
         }
+        sections = {
+            "Required": (
```

Comment:
I don't think "Required" is the right way to mark this section. Consider alternatives like "" (to designate uncollapsible fields), "Main", or maybe something else.

This is because the fields in this section are not so much "required" but instead important/useful for assignments.